molgenis-settings/src/main/java/org/molgenis/settings/entity/EntitySettingsPackage.java

```diff
@@ -0,0 +1,36 @@
+package org.molgenis.settings.entity;
+
+import static java.util.Objects.requireNonNull;
+import static org.molgenis.data.system.model.RootSystemPackage.PACKAGE_SYSTEM;
+
+import org.molgenis.data.meta.SystemPackage;
+import org.molgenis.data.meta.model.PackageMetadata;
+import org.molgenis.data.system.model.RootSystemPackage;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+@Component
+public class EntitySettingsPackage extends SystemPackage {
+  public static final String SIMPLE_NAME = "entity";
```

Comment:
**chore:** Use a two or three letter abbreviation here.

Because of restrictions on name length in PostgreSQL and Elasticsearch, fully qualified entity type names need to be shortened sometimes. To keep things readable, we use two or three letter package names so the EntityType's name doesn't get cut off (completely).

Reply:
changed to 'ent' 